### ISSUE TITLE
Remove polyfill.io dependency

### ISSUE THREAD

**Issue by mind84** [qgis2web/qgis2web#1102]
Hi!

I noticed that the main `index.html` project page includes the `polyfill` script as external resource.

https://github.com/qgis2web/qgis2web/blob/2524210e691947cccae059b2f65586501797732e/olFileScripts.py#L55

According to [this](https://fossa.com/blog/polyfill-supply-chain-attack-details-fixes/) article (and many others) `ployfill` dependency should be removed from source code.

The link is now broken, so the resource is no longer available anyway.

No problem for me to make a PR.

If you were already planning on removing it then I apologize for the noise.

Thank you

**Comment by andreaordonselli** [comment]
Thanks for the report, the openlayers export looks at local polyfills.js, I didn't realize that leaflet still looked at the web version. Thanks, I'll take care of it in the next release.

**Comment by andreaordonselli** [comment]
Question: Do you think it is still necessary to use polyfill nowadays?

**Comment by mind84** [comment]
hi @andreaordonselli,

DISCLAIMER:
this is my personal opinion :)

TLDR;
**No**

IN DETAIL:
I've never used it, at least in explicit manner. 

I haven't taken a closer look at the js code generated by the plugin so I cannot say if you have some functionality which requires backwards compatibility or else. But I think that, beside external js dependencies (that should handle polyfill by themselves "natively" or by other dependencies ), you can do all the job without it and fill any gap you find by writing your own js code.

**Comment by tomchadwin** [comment]
My best guess is that polyfill.io is used because at the time, the OpenLayers examples heavily used it. I strongly agree now with @mind84 that there is no overall need, and if there are specific polyfills still required, we'll become aware as people report failures.

**Comment by andreaordonselli** [comment]
Perfect, I'll remove it. I had already carried out tests a few months ago and had not encountered any problems without polyfills
